### ISSUE TITLE
ERROR - Plugin class loader for connector: was not found.

### ISSUE THREAD

**Issue by valdar** [apache/camel-kafka-connector#118]
Tests logs have this error:

```
2020-04-03 17:10:52,295 [main           ] ERROR org.apache.kafka.connect.runtime.isolation.DelegatingClassLoader - Plugin class loader for connector: 'org.apache.camel.kafkaconnector.CamelSinkConnector' was not found. Returning: org.apache.kafka.connect.runtime.isolation.DelegatingClassLoader@78aa7b77
```

it seems to 

> indicates that the connector is getting loaded in the same classloader as the connect worker, probably because the connector is in the Kafka lib directory. I would suggest moving the connector out of that directory to avoid this exception and have it loaded only through the `plugin.path` instead.
(see https://github.com/confluentinc/kafka-connect-jdbc/issues/766)

it seems not to be an issue for the tests themselves, still better to investigate.

**Comment by valdar** [comment]
@orpiske any idea on this one?

**Comment by orpiske** [comment]
@valdar I think that makes sense and it goes in line with the warnings I saw when building the containers for this. It doesn't seem to affect the test, but it would be better and safer not to have it. I will have time next week and will take a look at how we can get rid of it.

**Comment by orpiske** [comment]
@valdar it might be a tricky one to solve, though. I think we will need to find a way to isolate the [embedded runner](https://github.com/apache/camel-kafka-connector/blob/master/tests/src/test/java/org/apache/camel/kafkaconnector/services/kafkaconnect/KafkaConnectRunnerService.java) from the rest of the test code so that the runner itself is not on the same class loader. I don't really know how this black magic would work, so I'll have to research.

**Comment by valdar** [comment]
@orpiske tnx!

**Comment by valdar** [comment]
fixed with #138
